### ISSUE TITLE
Incomplete searchlight demo on website [Documentation]

### ISSUE THREAD

**Issue by piomonti** [nilearn/nilearn#2236]
Hi, 
I am currently trying to work through Section 2.4 " Searchlight : finding voxels containing information" which is available online here:
https://nilearn.github.io/decoding/searchlight.html#id5

I am working my way through this demo in order to familiarise myself with nilearn but it seems some steps are missing. 
In particular, in section 2.4.5.2:
 - a (previously undefined) object **p_unmasked** is introduced when running SPM
 - also **grouped_conditions_encoded** and **grouped_fmri_masked** when running mass univariate tests 

As such, when I try to the code provided I get a ```NameError: name 'p_unmasked' is not defined``` 

I am running nilearn version 0.5.2

**Comment by KamalakerDadi** [comment]
Hi @piomonti 

Documentation provides big picture about the methods unlike in an example. I think if you click on searchlight output image, it should point to a full working example. There it explains step-by-step implementation on simple dataset.

Hope this helps.

**Comment by bthirion** [comment]
Thx for reporting.
But we should make the link more explicit: clicking on the image to get the script is not clear enough imho.

**Comment by kchawla-pi** [comment]
Yes it is not clear enough. I only stumbled on to it by accident.
I'll open a PR.
Kshitij

------------------------------------------------------------
"Those who never wander might be lost."

Working to be kind, aware, equanimous, and skilled at python.

**Comment by piomonti** [comment]
@KamalakerDadi ah makes sense now! Thanks :) !

**Comment by emdupre** [comment]
Was this resolved with #2257  ?
